drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/editor/GuidedDecisionTableEditorPresenter.java

```diff
@@ -179,35 +181,45 @@ public void onClose() {
     }
 
     @Override
-    protected void onDecisionTableSelected( final @Observes DecisionTableSelectedEvent event ) {
-        super.onDecisionTableSelected( event );
+    protected void onDecisionTableSelected(final @Observes DecisionTableSelectedEvent event) {
+        super.onDecisionTableSelected(event);
     }
 
     @Override
     public void makeMenuBar() {
         this.menus = fileMenuBuilder
-                .addSave( getSaveMenuItem() )
-                .addCopy( () -> getActiveDocument().getCurrentPath(),
-                          fileNameValidator )
-                .addRename( () -> getActiveDocument().getLatestPath(),
-                            fileNameValidator )
-                .addDelete( () -> getActiveDocument().getLatestPath() )
-                .addValidate( () -> onValidate( getActiveDocument() ) )
-                .addNewTopLevelMenu( getEditMenuItem() )
-                .addNewTopLevelMenu( getViewMenuItem() )
-                .addNewTopLevelMenu( getInsertMenuItem() )
-                .addNewTopLevelMenu( getRadarMenuItem() )
-                .addNewTopLevelMenu( getVersionManagerMenuItem() )
+                .addSave(getSaveMenuItem())
+                .addCopy(() -> getActiveDocument().getCurrentPath(),
+                         fileNameValidator)
+                .addRename(() -> getActiveDocument().getLatestPath(),
+                           fileNameValidator)
+                .addDelete(() -> getActiveDocument().getLatestPath())
+                .addValidate(() -> onValidate(getActiveDocument()))
+                .addNewTopLevelMenu(getEditMenuItem())
+                .addNewTopLevelMenu(getViewMenuItem())
+                .addNewTopLevelMenu(getInsertMenuItem())
+                .addNewTopLevelMenu(getRadarMenuItem())
+                .addNewTopLevelMenu(getVersionManagerMenuItem())
                 .build();
     }
 
     @Override
-    public void onOpenDocumentsInEditor( final List<Path> selectedDocumentPaths ) {
+    public void onOpenDocumentsInEditor(final List<Path> selectedDocumentPaths) {
         throw new UnsupportedOperationException();
     }
 
     @Override
-    public void getAvailableDocumentPaths( final Callback<List<Path>> callback ) {
-        callback.callback( Collections.<Path>emptyList() );
+    public void getAvailableDocumentPaths(final Callback<List<Path>> callback) {
+        callback.callback(Collections.<Path>emptyList());
+    }
+
+    @Override
+    public void removeDocument(GuidedDecisionTableView.Presenter dtPresenter) {
```

Comment:
Here's the change. This method is called from ```KieMultipleDocumentEditor``` when the "delete" menu item is clicked.